docs/architecture/adr-050-sign-mode-textual.md

```diff
@@ -167,29 +168,29 @@ We define "transaction envelope" as all data in a transaction that is not in the
 ```
 Chain ID: <string>
 Account number: <uint64>
-*Public Key: <hex_string>
 Sequence: <uint64>
-<TxBody>                                                    // See #8.
-Fee: <coins>                                                // See value renderers for coins encoding.
-*Fee payer: <string>                                        // Skipped if no fee_payer set
-*Fee granter: <string>                                      // Skipped if no fee_granter set
-Memo: <string>                                              // Skipped if no memo set
+*Public Key: <Any>
+This transaction has <int> Message
+> Message (<int>/<int>): <Any>                              // See value renderers for Any rendering.
+End of Message
+Memo: <string>                                              // Skipped if no memo set.
+Fee: <coins>                                                // See value renderers for coins rendering.
+*Fee payer: <string>                                        // Skipped if no fee_payer set.
+*Fee granter: <string>                                      // Skipped if no fee_granter set.
+Tip: <coins>                                                // Skippted if no tip.
+Tipper: <string>
 *Gas Limit: <uint64>
-*Timeout Height:  <uint64>                                  // Skipped if no timeout_height set
-Tipper: <string>                                            // If there's a tip
-Tip: <string>
-*This transaction has <int> body extension:                 // Skipped if no body extension options
-*<repeated Any>
-*This transaction has <int> body non-critical extensions:   // Skipped if no body non-critical extension options
-*<repeated Any>                                             // See value renderers for Any and array encoding.
-*This transaction has <int> body auth info extensions:      // Skipped if no auth info extension options
-*<repeated Any>
-*This transaction has <int> other signers:                  // Skipped if there is only one signer
-*Signer (<int>/<int>):
-*Public Key: <hex_string>
-*Sequence: <uint64>
+*Timeout Height: <uint64>                                   // Skipped if no timeout_height set.
+*Other signer: <int> SignerInfo                             // Skipped if the transaction only has 1 signer.
+*> Other signer (<int>/<int>): <SignerInfo>
 *End of other signers
-*Hash of raw bytes: <hex_string>                            // Hex encoding of bytes defined in #10, to prevent tx hash malleability.
+*Extension options: <int> Any:                              // Skipped if no body extension options
+*> Extension options (<int>/<int>): <Any>
+*End of extension options
+*Non critical extension options: <int> Any:                 // Skipped if no body non critical extension options
+*> Non critical extension options (<int>/<int>): <Any>
+*End of Non critical extension options
+*Hash of raw bytes: <hex_string>                            // Hex encoding of bytes defined, to prevent tx hash malleability.
```

Comment:
What do you think about calling this field just "TxHash"?

Reply:
I think calling it TxHash is more confusing. TxHash is a commonly used term, referring to the hash of [TxRaw](https://github.com/cosmos/cosmos-sdk/blob/a1dc6d999809a9f5a334edae49637c3522bf66ae/proto/cosmos/tx/v1beta1/tx.proto#L33). But TxRaw includes the signatures, which by definition we don't have at this step.

I can't think of a word for "Tx, but without the sigs", but I'm also not 100% satisfied with "raw bytes". Maybe "transaction content"?

Reply:
If there's no existing term, then what you have is fine. It's an expert field after all.